README.adoc

```diff
@@ -6,6 +6,7 @@ endif::[]
 // URIs:
 :uri-letsencrypt: https://letsencrypt.org
 :uri-letsencrypt-article: https://fedoramagazine.org/letsencrypt-now-available-fedora/
+:uri-awestruct: https://www.staticgen.com/awestruct
```

Comment:
Let's link to the source repository instead. https://github.com/awestruct/awestruct